src/reference/sectionB_elements.inc

```diff
@@ -252,26 +254,30 @@ A :code:`variable` element information item (referred to in this specification a
    .. container:: issue-2-8-1-1
 
       1. The :code:`name` attribute.
-         The value of the :code:`name` attribute MUST be a :ref:`CellML identifier<specA_cellml_identifier>`.
+      
+         1. The value of the :code:`name` attribute MUST be a :ref:`CellML identifier<specA_cellml_identifier>`.
```

Comment:
The main reason is because restructured text doesn't maintain the numbering if you enter a directive like `container` in between numbered list items.

Reply:
Ok, I thought it might have been something like that and maybe one more reason for only using `1.` and the like rather than `#.`? Anyway, just a preference, so will "resolve" this conversation.